### ISSUE TITLE
I can't setup the local environment in windows

### ISSUE THREAD

**Issue by GermanJablo** [nextauthjs/next-auth#7942]
### Environment

  System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Memory: 13.78 GB / 31.77 GB
  Binaries:
    Node: 18.13.0 - C:\Program Files\nodejs\node.EXE
    npm: 8.19.3 - C:\Program Files\nodejs\npm.CMD
    pnpm: 7.23.0 - C:\Program Files\nodejs\pnpm.CMD
  Browsers:
    Edge: Spartan (44.22621.1848.0), Chromium (114.0.1823.67)
    Internet Explorer: 11.0.22621.1

### Reproduction URL

https://github.com/nextauthjs/next-auth

### Describe the issue

I follow the instructions in "[Setting up local environment](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md#setting-up-local-environment)" but when I run `npm run dev` I get:

> Module not found: Can't resolve 'next-auth/react'

attached video:

https://www.loom.com/share/a1d0cc94b0474daf8b0302199cc20437

In Reproduction URL I have put the next-auth repository. If you need it, I can do a fork where node_modules is not gitignored.

### How to reproduce

Basically follow the steps in the guide "[Setting up local environment](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md#setting-up-local-environment)" in Windows.

The only difference is that I used `ren` instead of `cp` to rename `.env.local.example` because I'm using `cmd`. I have confirmed that the name change was successful.

### Expected behavior

`pnpm run dev` should run the development app on localhost:3000

**Comment by balazsorban44** [comment]
I cannot reproduce this. Check out https://github.com/nextauthjs/next-auth/issues/7934#issuecomment-1616886937 which miggt help. 🤔
